package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "3.53.1",
```

Comment:
Do we need to bump this now when we modify one of our components? I didn't know.